src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx

```diff
@@ -115,6 +122,7 @@ const baseAppConfigs: ApplicationConfig[] = [
   },
   {
     name: 'Reactive Analytics',
+    displayName: 'Reactive Analytics',
```

Comment:
Reactive Analytics should have environments too, but we can address this separately later.